nasa-gibs/worldview
# WV-2124: Bottom of Vector Modal cutoff when changing orientation in mobile view

## Description

The initial render of a vector modal in either portrait or landscape mode on a mobile device works properly but an orientation change of the device (landscape --> portrait or portrait --> landscape) causes the modal to be improperly sized for the viewport, obscuring data from the user. This PR addresses the modal width/height properties following an orientation change to ensure the user can access all of the data in the modal regardless of orinetation.

## How To Test
- Load a clean WV instance.
- Open devtools & enter mobile mode.
- Select either portrait or landscape orientation (please test starting from each orientation)
- Expand the Layers menu.
- Click the orange filter button, scroll to the bottom & select Vector & click Apply.
- Select any vector layer displayed & click Add Layer.
- Close the window & collapse the Layers menu so you can see the map.
- Zoom in on any red dot on the map & click it to trigger the modal.
- Confirm the modal is appropriately sized & positioned in the window.
- Change the orientation of the device & re-confirm the height & positioning of the modal remains appropriate.

## Additional Notes
- This PR should be tested directly on mobile devices; we should not rely solely on the Chrome dev tools to confirm this works as intended.
- There are other Modal instances in the codebase which should be reviewed to confirm this fix did not negatively impact them. 

@nasa-gibs/worldview
